### ISSUE TITLE
Add Kotlin-Language-Server(Kotlin Auto Completion)

### ISSUE THREAD

**Issue by yz-gm** [AndroidIDEOfficial/AndroidIDE#1239]
### Feature description

Please take a look at this open source address: https://github.com/fwcd/kotlin-language-server

### What version of AndroidIDE you're using?

v2.5.2 (debug builds)

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by itsaky** [comment]
Duplicate of #211.

There are a lot of things to implement/fix before we start working on this.
